v-spec.adoc

```diff
@@ -1132,15 +1132,23 @@ the `vill` bit is set in `vtype`, the remaining bits in `vtype` are
 set to zero, and the `vl` register is also set to zero.
 
 The requested application vector length (AVL) is passed in `rs1` as an
-unsigned integer.  Using `x0` as the `rs1` register specifier, encodes
-an infinite AVL, and so requests the maximum possible vector length.
+unsigned integer.  Using `x0` as the `rs1` register specifier requests
+an AVL equal to the current `vl`.
 
-NOTE: A `vsetvl{i}` with AVL=`x0` can be used to read current VLMAX,
-though this does overwrite `vl`.
+NOTE: Executing `vsetvl{i}` with `rs1`=`x0` is guaranteed to keep the
+current `vl` intact if VLMAX is not reduced.
```

Comment:
`vl` is not reset and it can have arbitrary value before 1st `vsetvli`

Reply:
"The vector extension must have a consistent state at reset" -- so `vl` can be arbitrary only up to the point that it remains valid for the reset value of `vtype`, in which case this statement still holds.

Reply:
(And of course if `vtype.vill` is high out of reset, then `vl` must appear to be zero, in which case this statement holds, too.)

Reply:
The note says it is only recommended to set `vl` to zero at reset

Reply:
Yeah, but as written, the statement "The vector extension must have a consistent state at reset" is a requirement.